wagtail_localize/test/models.py

```diff
@@ -43,6 +45,19 @@ class TestSnippet(TranslatableMixin, models.Model):
     ]
 
 
+class TestUUIDModel(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    charfield = models.CharField(max_length=10, blank=True)
+
+
+@register_snippet
+class TestUUIDSnippet(TranslatableMixin, models.Model):
+    # id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Stray bit of commented out code here. If it doesn't need to be here lets drop it.